File: 프로그래머스/unrated/181902. 문자 개수 세기/문자 개수 세기.cs
using System;

public class Solution {
    public int[] solution(string my_string)
    {
        int[] answer = new int[52];
        
        foreach(char c in my_string)
        {
            if(Char.IsUpper(c))
                answer[c - 'A']++;
            else
                answer[26 + c - 'a']++;
        }
        
        return answer;
    }
}

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(string my_string) {
    vector<int> answer;
    
    for (char i = 'A'; i <= 'Z'; ++i)
    {
        int num = std::count(my_string.begin(), my_string.end(), i);
        answer.push_back(num);
    }    
    for (char i = 'a'; i <= 'z'; ++i)
    {
        int num = std::count(my_string.begin(), my_string.end(), i);
        answer.push_back(num);
    }
    
    return answer;
}